.travis.yml

```diff
@@ -0,0 +1,25 @@
+---
+language: ruby
+sudo: false
+cache: bundler
+script:
+- bundle exec rake
+rvm:
+- 2.5.3
```

Comment:
Any reason to choose 2.5.3 here instead of `'2.5'` so it's the latest 2.5?

Reply:
2.5.3 is bundled in the travis image. so the run is a bit faster. if we set it to 2.5, travis will download the ruby version on each run.